core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -2019,7 +2019,7 @@ private[spark] class DAGScheduler(
           val ignoreStageFailure = ignoreDecommissionFetchFailure &&
             isExecutorDecommissioningOrDecommissioned(taskScheduler, bmAddress)
           if (ignoreStageFailure) {
-            logInfo(log"Ignoring fetch failure from ${MDC(TASK_ID, task)} of " +
+            logInfo(log"Ignoring fetch failure from ${MDC(TASK_NAME, task)} of " +
```

Comment:
Do we really want to call this `TASK_NAME`?
From the code below, it seems inconsistent:
https://github.com/apache/spark/blob/33a9c5d7f478a476fe9882ad8fe101fd60756a98/core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala#L839
https://github.com/apache/spark/blob/33a9c5d7f478a476fe9882ad8fe101fd60756a98/core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala#L583-L587

Reply:
well, I can't find a better name here..